controller/scope.go

```diff
@@ -0,0 +1,10 @@
+package controller
+
+import (
+	"context"
+)
+
+// GetWriteableGithubScopes creates a project specified by the project request, error otherwise
```

Comment:
```suggestion
// GetWritableGithubScopes creates a project specified by the project request, error otherwise
```

Reply:
Damn, how'd  you spot that one?

Reply:
I actually searched for "WritableGithubScopes" and also misspelled it, and realized that there was only a comment for this

Total luck really haha

Reply:
😆